explainer.md

```diff
@@ -304,8 +304,25 @@ body {
 ```
 ## Considered Alternatives
 
-### `getTitleBarAreaRect` method
-The `windowControlsOverlay` object has a `getBoundingClientRect` method that returns the area of the title bar where is it safe to put content. We consider naming the `windowControlsOverlay.getBoundingClientRect` method to `getTitleBarAreaRect` to remove any ambiguity of the area returned by the method. 
+### Treating the overlay as a notch
+
+Following the pattern of safe-area-inset-*s, we propose new CSS environment variables to define the insets of the unsafe notch area in more detail: 
```

Comment:
"we propose" or "we propose*d*"? Since this is a rejected alternative, should this be past tense?